bundles/model/org.eclipse.smarthome.model.rule/src/org/eclipse/smarthome/model/rule/jvmmodel/RulesJvmModelInferrer.xtend

```diff
@@ -29,112 +31,121 @@ import org.slf4j.LoggerFactory
  */
 class RulesJvmModelInferrer extends ScriptJvmModelInferrer {
 
+    static private final Logger logger = LoggerFactory.getLogger(RulesJvmModelInferrer)
```

Comment:
Don't know anything about Xtend / Xtext, but does we need to use a static variable here?

But you should use the common modifier order: private static final (or drop static).